### PR TITLE
fixes #75

### LINKED ISSUE #75: update example demo project to support supplying the i18n resources
It will be useful if we can pass i18n messages similar how we can change the currently displayed form using schema, uischema,data to see the final result

### PR DESCRIPTION
**Pull request by kchobantonov**
Fixes #75

### CODE REVIEW

**Comment by kchobantonov** [comment]
@sdirix please review

**Comment by kchobantonov** [comment]
@sdirix  I also added the ability to translate the dropdown labels

**Comment by sdirix** [comment]
Thanks for the PR :heart: , I'll take a look

**Review by lucas-koehler — CHANGES_REQUESTED** [review]
I added some minor comments.
Something seems to be broken with the enums. See the enum example: https://deploy-preview-76--jsonforms-vuetify-renderers.netlify.app/#/example/enum
If you open the console you can also see errors

File: example/src/i18n/i18n.ts [hunk 1]
```diff
@@ -1,91 +1,20 @@
...
 export const createTranslator =
-  (locale: 'en' | 'de') =>
+  (locale: string, translations?: Record<string, any>) =>
   (key: string, defaultMessage: string | undefined): string | undefined => {
-    return get(locale === 'en' ? en : de, key) ?? defaultMessage;
+    if (!translations) {
+      return defaultMessage;
+    }
+
+    let localeTraslations = translations[locale];
```
[the remark this answers is not in the document]
Comment (kchobantonov) [reply example/src/i18n/i18n.ts]: @lucas-koehler thanks  - fixed in the next commit

**Comment by kchobantonov** [comment]
> I added some minor comments. Something seems to be broken with the enums. See the enum example: https://deploy-preview-76--jsonforms-vuetify-renderers.netlify.app/#/example/enum If you open the console you can also see errors. It doesn't seem to find the translation function to translate the item labels:
> 
> ```
> vue.runtime.esm.js:3020 TypeError: e.t is not a function
>     at item-text (app.c5b53e55.js:1:118544)
> ```

thanks for catching this. I will fix in the next commit

**Comment by kchobantonov** [comment]
@lucas-koehler  please review again - https://deploy-preview-76--jsonforms-vuetify-renderers.netlify.app/#/example/enum

**Review by lucas-koehler — APPROVED** [review]
@kchobantonov Thanks for the updates, LGTM now :)

**Pull request merged**